tools/sgen/input/FluentUIStyle.yml

```diff
@@ -44,6 +44,23 @@ Colors:
     - brandSelected: "FluentUIColor(light: $Colors.Brand.shade20, dark: $Colors.Brand.shade20)"
     - brandDisabled: "FluentUIColor(light: NamedColor(grey74), dark: NamedColor(grey36))"
 
+  Notification:
```

Comment:
I don't think we need to define this generic section here unless we had a standard for colors in multiple controls that do some sort of notification. You can add these colors directly where they're used below.